core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -208,6 +235,22 @@ void setServerConfig(ServerConfig serverConfig) {
         router.registerMetrics(registry, idPrefix);
     }
 
+    /**
+     * Sets the {@link Logger} which is used for writing access logs of this virtual host.
+     */
+    void accessLogger(Logger logger) {
+        requireNonNull(logger, "logger");
+        accessLogger = logger;
```

Comment:
nit: can be simplified to `accessLogger = requireNonNull(logger, "logger");`

If you configured IntelliJ IDEA, you can auto-complete this really easily by typing `rnna`.